grails-app/migrations/views/drop-all-views.xml

```diff
@@ -38,6 +38,12 @@
       DROP VIEW IF EXISTS substitution_item;
       DROP VIEW IF EXISTS supplier;
       DROP VIEW IF EXISTS stock_movement;
+      DROP VIEW IF EXISTS order_item_status;
+      DROP VIEW IF EXISTS order_receipt_status;
+      DROP VIEW IF EXISTS order_payment_status_from_shipments;
+      DROP VIEW IF EXISTS order_payment_status_from_adjustments;
+      DROP VIEW IF EXISTS order_summary;
+      DROP VIEW IF EXISTS order_item_summary;
```

Comment:
All other views are above. The only one missing was the `order_item_summary` (add this one next to the others related above). No need to drop the other views twice ;)